icu4c/source/i18n/unicode/udisplayoptions.h

```diff
@@ -0,0 +1,316 @@
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_NEUTER = 2,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_FEMININE = 3,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_MASCULINE = 4,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_ANIMATE = 5,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_INANIMATE = 6,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_PERSONAL = 7,
+    /** ICU 72 */
+    UDISPOPT_NOUN_CLASS_COMMON = 8,
+} UDisplayOptionsNounClass;
+
+/**
+ * @return the lowercase CLDR keyword string for the noun class.
+ *
+ * @draft ICU 72
+ */
+U_CAPI const char * U_EXPORT2
+udispopt_getNounClassIdentifier(UDisplayOptionsNounClass nounClass);
+
+/**
+ *
+ * @param identifier in lower case such as "feminine" or "masculine"
+ * @return the plural category corresponding to the identifier, or `UDISPOPT_NOUN_CLASS_UNDEFINED`
+ *
+ * @draft ICU 72
+ */
+U_CAPI UDisplayOptionsNounClass U_EXPORT2
+udispopt_fromNounClassIdentifier(const char *identifier);
+
+/**
+ * Represents all the capitalization options.
+ *
+ * @draft ICU 72
+ */
+typedef enum UDisplayOptionsCapitalization {
+    /**
+     * A possible setting for Capitalization.
+     * The capitalization context to be used is unknown (this is the default value).
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_CAPITALIZATION_UNDEFINED = 0,
+
+    /**
+     * The capitalization context if a date, date symbol or display name is to be
+     * formatted with capitalization appropriate for the beginning of a sentence.
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_CAPITALIZATION_BEGINNING_OF_SENTENCE = 1,
+
+    /**
+     * The capitalization context if a date, date symbol or display name is to be
+     * formatted with capitalization appropriate for the middle of a sentence.
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_CAPITALIZATION_MIDDLE_OF_SENTENCE = 2,
+
+    /**
+     * The capitalization context if a date, date symbol or display name is to be
+     * formatted with capitalization appropriate for stand-alone usage such as an
+     * isolated name on a calendar page.
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_CAPITALIZATION_STANDALONE = 3,
+
+    /**
+     * The capitalization context if a date, date symbol or display name is to be
+     * formatted with capitalization appropriate for a user-interface list or menu item.
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_CAPITALIZATION_UI_LIST_OR_MENU = 4,
+} UDisplayOptionsCapitalization;
+
+/**
+ * Represents all the dialect handlings.
+ *
+ * @draft ICU 72
+ */
+typedef enum UDisplayOptionsNameStyle {
+    /**
+     * A possible setting for NameStyle.
+     * The NameStyle context to be used is unknown (this is the default value).
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_NAME_STYLE_UNDEFINED = 0,
+
+    /**
+     * Use standard names when generating a locale name,
+     * e.g. en_GB displays as 'English (United Kingdom)'.
+     *
+     * @draft ICU 72
+     */
+    UDISPOPT_NAME_STYLE_STANDARD_NAMES = 1,
```

Comment:
Suggestion: `UDISPOPT_NAME_STYLE_STANDARD` (remove the redundant "NAMES")

Reply:
lets approved it right now and discuss the change next week in the ICU meeting because it needs an approval from the committee.